### DISCUSSION TITLE
账号被封禁如何解决？

### DISCUSSION THREAD

**Discussion by bugwz** [BurnHair/docs#123] [技术支持]
![image](https://github.com/BurnHair/docs/assets/12412413/dbf3c8b1-fe1c-4bc7-8b82-b3cbf0ca7611)

**Comment by bugwz** [comment]
我的使用方式如下：

lobechat（部署在vercul）=> 我自己搭建的 oneapi（部署在 render） => burnhair服务

**Comment by BennyThink** [comment]
你被检测到了重复的会话，如果有共享 API Key的话就很容易这样。仅此一次下不为例

**Reply by bugwz** [reply]
我的oneapi上只有burnhar的我自己的一个key，所以应该不存在共享api key的情况。

我有几个疑问：

1. 我的 oneapi 搭建在 render 中，并且没有独立ip，这种情况下，如果其他人也会使用 render 的服务，是否会导致 burnhair 中看到的访问 ip 相同？这是否会误判用户。
2. 我在调用接口的时候，如果开了两个窗口接连调用也会被判断重复的会话吗？

@BennyThink

**Reply by BennyThink** [reply]
1. 有可能
2. 都是自己的 key 就没关系的
